scripts/testIOS.sh

```diff
@@ -4,8 +4,9 @@ THISDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 SIMNAME=$1
 SIMVER=$2
+ACTION=$3
 
 echo Testing ${SIMNAME} with iOS ${SIMVER}...
 
 cd $THISDIR/../ios
-set -o pipefail && xcodebuild -scheme Buttercup -destination "platform=iOS Simulator,name=${SIMNAME},OS=${SIMVER}" build test | xcpretty -c
+set -o pipefail && xcodebuild -UseModernBuildSystem=NO -scheme Buttercup -destination "platform=iOS Simulator,name=${SIMNAME},OS=${SIMVER}" ${ACTION} | xcpretty -c
```

Comment:
Is `UseModernBuildSystem=NO` because of the module I mentioned or is there another reason?

Reply:
I set the build system to legacy, because the default build system is not running :/

Reply:
I removed it